spec/unit/gameSpec.js

```diff
@@ -0,0 +1,86 @@
+describe("Game", function () {
+  let game;
+  let frame;
+
+  beforeEach(function () {
+    frame = { 
+      addRoll: () => {},
+      addBonusScore: () => {},
+      calcFrameTotal: () => {},
+      setFrameTotal: () => {},
+      isFinalFrameComplete: () => {},
+      isFrameComplete: () => {},
+      isStrike: () => {},
+      isSpare: () => {}
+    };
+    spyOn(frameFactory, "createFrame").and.returnValue(frame);
+    game = new Game();
+  });
+
+  describe("new instancs of game class", function () {
+    it("frame array is empty for frame instance", function () {
+      expect(game.frames).toEqual([]);
+    });
+
+    it("have current frame set to frame passed in", function () {
+      expect(game.currentFrame).toEqual(frame);
+    });
+  });
```

Comment:
These seem like implementation/state tests. I don't think the current Frame should be part of the public interface.